<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetAllAdminUsers</class>
 <widget class="QDialog" name="GetAllAdminUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>571</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Name Of Admin User</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboAllAdminNameComm">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>431</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>571</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>User Info</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Category Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Email ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone No:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAllAdminCtgryNameComm">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblAllAdminEmailComm">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblAllAdminPhComm">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblAllAdminAddressComm">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnAllAdminCancelComm">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>480</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
